### PR TITLE
Submission

### PR DESCRIPTION
**Pull request by YourboiPrince**

### CODE REVIEW

File: app/Filament/Resources/BlogResource/Pages/CreateBlog.php
Comment (Edward-Kabue) [inline app/Filament/Resources/BlogResource/Pages/CreateBlog.php:1]: Add return to index on after user clicks on submit button

File: app/Filament/Resources/ContactResource/Pages/CreateContact.php
Comment (Edward-Kabue) [inline app/Filament/Resources/ContactResource/Pages/CreateContact.php:1]: Same to this

File: routes/web.php
Comment (Edward-Kabue) [inline routes/web.php:1]: Remove the laravel breeze routes as you are now using filament

**Review by Edward-Kabue — APPROVED** [review]
STATUS: Approved 

 @YourboiPrince 
This milestone is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application

**Comment by YourboiPrince** [comment]
Capture1
Capture2
Capture3
Capture4
Capture5
Capture6
Capture7

**Pull request merged**